### PR TITLE
Updated Visa icon

### PR DESCRIPTION
**Pull request by ymaksym**

### CODE REVIEW

**Review by simonsandersje — DISMISSED** [review]

File: icons/payment-visa.svg [hunk 1]
```diff
@@ -1,13 +1 @@
...
-    />
-    <path
-      d="M28.09 20.476h-7.925l-.065.395c6.167 1.58 10.247 5.395 11.94 9.98l-1.72-8.765c-.3-1.21-1.162-1.567-2.23-1.61"
-      fill="#ED982D"
-    />
-  </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 216 216"><switch><g><path d="M144.4 128.2c0 1.5-1.2 2.7-2.7 2.7H74.5c-1.5 0-2.7-1.2-2.7-2.7V87.8c0-1.5 1.2-2.7 2.7-2.7h67.2c1.5 0 2.7 1.2 2.7 2.7v40.4z" fill="#1a1f71"/><path d="M99.4 98.4l-8.1 19.3H86l-4-15.4c-.2-.9-.5-1.3-1.2-1.7-1.2-.6-3.2-1.3-4.9-1.6l.1-.6h8.5c1.1 0 2.1.7 2.3 2l2.1 11.2 5.2-13.1h5.3zm20.7 13c0-5.1-7-5.4-7-7.6 0-.7.7-1.4 2.1-1.6.7-.1 2.7-.2 4.9.9l.9-4.1c-1.2-.4-2.7-.9-4.7-.9-4.9 0-8.4 2.6-8.4 6.4 0 2.8 2.5 4.3 4.4 5.2 1.9.9 2.6 1.5 2.6 2.4 0 1.3-1.5 1.9-3 1.9-2.5 0-4-.7-5.1-1.2l-.9 4.2c1.2.5 3.3 1 5.5 1 5.2 0 8.6-2.6 8.7-6.6m13 6.3h4.6l-4-19.3h-4.3c-1 0-1.8.6-2.1 1.4l-7.5 17.9h5.2l1-2.9h6.4l.7 2.9zm-5.6-6.8l2.6-7.2 1.5 7.2h-4.1zm-21-12.5l-4.1 19.3h-5l4.1-19.3h5z" fill="#fff"/></g></switch></svg>
```
Comment (DamianMullins) [inline icons/payment-visa.svg:1]: I'm not sure the viewbox dimensions are correct here? As they stand there is a large amount of space surrounding the icon?

**Comment by 1itvinka** [comment]
have you used any svg optimisation tool like https://jakearchibald.github.io/svgomg/?

**Comment by ymaksym** [comment]
Hi @1itvinka  - I got this svg file from UX guys - do you think I should also put it though the svgomg tool?

**Comment by 1itvinka** [comment]
@ymaksym It will drastically reduce the file size. Not sure if it's done automatically on build. But ideally we should always optimise images before adding them to the projects.

**Comment by ymaksym** [comment]
@1itvinka will use svgomg for it

**Review by 1itvinka — DISMISSED** [review]

**Review by charliearlie — DISMISSED** [review]

File: icons/payment-visa.svg [hunk 2]
```diff
@@ -1,13 +1,4 @@
...
-    <path
-      fill="#FFF"
-      d="M41.49 39.547l3.075-19.074h4.915l-3.075 19.074H41.49M64.233 20.94c-.972-.382-2.5-.8-4.404-.8-4.86 0-8.28 2.59-8.31 6.295-.03 2.742 2.442 4.27 4.305 5.182 1.915.935 2.56 1.532 2.55 2.368-.012 1.277-1.528 1.862-2.94 1.862-1.968 0-3.014-.288-4.63-1l-.632-.304-.688 4.27c1.146.53 3.27.992 5.476 1.017 5.165 0 8.52-2.558 8.56-6.516.018-2.174-1.292-3.822-4.128-5.182-1.718-.885-2.772-1.47-2.76-2.366 0-.792.89-1.64 2.816-1.64a8.605 8.605 0 0 1 3.678.732l.44.218.667-4.135m6.544 11.843c.408-1.096 1.958-5.33 1.958-5.33-.028.05.405-1.106.653-1.82l.332 1.644s.943 4.553 1.138 5.506h-4.08zm6.064-12.292h-3.798c-1.178 0-2.06.338-2.576 1.58l-7.3 17.482h5.164s.842-2.352 1.033-2.868c.563 0 5.58.01 6.295.01.147.665.6 2.858.6 2.858h4.56L76.84 20.492zm-39.474-.002l-4.81 13.003-.516-2.644c-.896-3.046-3.686-6.347-6.807-8.002l4.4 16.682 5.202-.003 7.74-19.038h-5.208"
-    />
-    <path
-      d="M28.09 20.476h-7.925l-.065.395c6.167 1.58 10.247 5.395 11.94 9.98l-1.72-8.765c-.3-1.21-1.162-1.567-2.23-1.61"
-      fill="#ED982D"
-    />
-  </g>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 72.6 45.8" xml:space="preserve">
```
Comment (DamianMullins) [inline icons/payment-visa.svg:1]: You can safely remove some of the attributes here as they aren't required

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 72.6 45.8">
```

Comment (ymaksym) [reply icons/payment-visa.svg]: Done, cheers @DamianMullins

**Review by 1itvinka — APPROVED** [review]

**Review by oleksiishevchukje — APPROVED** [review]

**Review by DamianMullins — APPROVED** [review]

**Pull request merged**